todoman/formatters.py

```diff
@@ -82,7 +82,6 @@ def detailed(self, todo):
             extra_rows += self._columnize('Description', todo.description)
         if todo.location:
             extra_rows += self._columnize('Location', todo.location)
-
```

Comment:
This particular change is the removal of an empty line. It does not alter any functionality, and the change is completely unrelated to this PR.